compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -624,7 +624,8 @@ expression
     |   awaitExpression                                                     # awaitExprExpression
     |	expression matchExpression										    # matchExprExpression
     |   expression ELVIS expression                                         # elvisExpression
-    |   expression (BITAND | PIPE | BITXOR) expression                      # bitwiseExpression
+    |   expression (BIT_AND | PIPE | BIT_XOR) expression                    # bitwiseExpression
+    |   (BIT_COMPLEMENT) expression                                         # bitwiseComplementExpression
```

Comment:
~ expression should have higher precedence than others. For example, ~ 1 & 2 passed as (~1) & 2. not ~ (1 & 2)